tools/yaml-templates/ios-test.yml

```diff
@@ -18,7 +18,10 @@ steps:
       targetType: inline
       script: |
         brew install xctesthtmlreport
+
+        echo "##vso[task.setvariable variable=xctestHtmlReportInstalled]Yes"
```

Comment:
Can you add a comment or two explaining what this does and why it is here?

Reply:
Comments added. Please take a look.